### ISSUE TITLE
Release 2.0.0 does not work on OS X

### ISSUE THREAD

**Issue by take-five** [dalibo/pg_activity#165]
Today I updated `pg_activity`: `pip3 install pg_activity --upgrade`.

When I try to launch it, I get this error:
```
$ pg_activity
Traceback (most recent call last):
  File "/usr/local/bin/pg_activity", line 295, in <module>
    ui.main(term, dataobj, host, options, dsn)
  File "/usr/local/lib/python3.8/site-packages/pgactivity/ui.py", line 165, in main
    memory, swap, load = activities.mem_swap_load()
  File "/usr/local/lib/python3.8/site-packages/pgactivity/activities.py", line 231, in mem_swap_load
    mem_swap = get_mem_swap()
  File "/usr/local/lib/python3.8/site-packages/pgactivity/activities.py", line 223, in get_mem_swap
    buffers = phymem.buffers
AttributeError: 'svmem' object has no attribute 'buffers'
```

OS X version: 10.14.6
Python version: 3.8.5
pip version: 20.1.1

**Comment by anthonydb** [comment]
Good day. I've encountered this issue as well. I use `peen` for Python management.

macOS version: 10.15.7
Python version: 3.9.1

```
Traceback (most recent call last):
  File "/Users/xxxxx/.pyenv/versions/data-3.9.1/bin/pg_activity", line 295, in <module>
    ui.main(term, dataobj, host, options, dsn)
  File "/Users/xxxxx/.pyenv/versions/3.9.1/envs/data-3.9.1/lib/python3.9/site-packages/pgactivity/ui.py", line 165, in main
    memory, swap, load = activities.mem_swap_load()
  File "/Users/xxxxx/.pyenv/versions/3.9.1/envs/data-3.9.1/lib/python3.9/site-packages/pgactivity/activities.py", line 231, in mem_swap_load
    mem_swap = get_mem_swap()
  File "/Users/xxxxx/.pyenv/versions/3.9.1/envs/data-3.9.1/lib/python3.9/site-packages/pgactivity/activities.py", line 223, in get_mem_swap
    buffers = phymem.buffers
AttributeError: 'svmem' object has no attribute 'buffers'
```

**Comment by nico0olas** [comment]
me too, I try on docker ubuntu and centos7 (from macos) but it's not good too

Traceback (most recent call last):
  File "./pg_activity", line 295, in <module>
    ui.main(term, dataobj, host, options, dsn)
  File "/pg_activity/pgactivity/ui.py", line 215, in main
    views.screen(
  File "/pg_activity/pgactivity/views.py", line 520, in screen
    processes_rows(
  File "/pg_activity/pgactivity/views.py", line 124, in wrapper
    for line in func(term, *args, **kwargs):
  File "/pg_activity/pgactivity/views.py", line 404, in processes_rows
    query_lines = [wrapped_lines[0]] + term.wrap(
IndexError: list index out of range

**Comment by dlax** [comment]
Thanks for reporting. I just released and uploaded [version 2.0.2](https://github.com/dalibo/pg_activity/releases/tag/v2.0.2) which should fix these issues, can you try it out?

**Comment by take-five** [comment]
Thank you for quick reaction, version 2.0.2 fixed the problem.
